Kitodo/src/main/webapp/pages/indexingPage.xhtml

```diff
@@ -0,0 +1,209 @@
+<!--
+ *
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ *
+-->
+
+<ui:composition
+        template="/WEB-INF/templates/base.xhtml"
+        xmlns:h="http://xmlns.jcp.org/jsf/html"
+        xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+        xmlns:p="http://primefaces.org/ui"
+        xmlns:o="http://omnifaces.org/ui">
+    <ui:define name="content">
+        <p:panel id="listWrapper">
+
+            <script type="text/javascript">
```

Comment:
Is inlining JavaScript a good idea? Would it not be better if this JavaScript is outside in a separate file?